azure-pipelines.yml

```diff
@@ -136,6 +153,22 @@ jobs:
       publishVstsFeed: '2cac8390-d7aa-4320-b481-1a32c53de85b'
       allowPackageConflicts: true
 
+  - task: PowerShell@1
+    condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+    displayName: 'Push VSIX'
+    inputs:
+      scriptType: inlineScript
+      inlineScript: |
+        # Setting these environment variables because they are used in the script for the upload endpoint: 
+     
+        $env:APPVEYOR_REPO_PROVIDER = "github"
+        $env:APPVEYOR_REPO_NAME = "microsoft/dotnet-apiport"
```

Comment:
Would this be a good time to rename these env vars to something more current? 
In fact, you don't actually need them at all, since both of these values are [predefined variables](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml).

Old | New
--|--
`$env:APPVEYOR_REPO_PROVIDER` | `$(Build.Repository.Provider)`
`$env:APPVEYOR_REPO_NAME` | `$(Build.Repository.Name)`

Reply:
Good to know. Fixed